src/panels/lovelace/cards/energy/hui-energy-compare-card.ts

```diff
@@ -33,6 +34,11 @@ export class HuiEnergyCompareCard
   // eslint-disable-next-line lit/no-native-attributes
   @property({ type: Boolean, reflect: true }) hidden = true;
 
+  // Energy compare card cannot tolerate being removed from the DOM by hui-card,
+  // as it calculates its own visibility and needs an active collection
+  // subscription to do so.
+  disconnectOnHide = false;
```

Comment:
I don't like this prop name. Maybe `removeOnHide`? Or `remainWhileHidden = true`? `connectedWhileHidden`?
In general it's better for the new and rare behavior to be described with `true`.

Reply:
Updated to your suggestion, but will maybe need to wait and see if Bram suggest another approach.